$OpenBSD: patch-qtdeclarative_src_qml_jsruntime_qv4stringobject_cpp,v 1.2 2015/04/19 13:18:28 zhuk Exp $
Unbreak compilation for C++11 mode (GCC 4.8.6 bug?).
Should go upstream probably anyway, though.
--- qtdeclarative/src/qml/jsruntime/qv4stringobject.cpp.ports.orig	Tue Feb 17 07:57:27 2015
+++ qtdeclarative/src/qml/jsruntime/qv4stringobject.cpp	Wed Apr  8 00:38:31 2015
@@ -37,6 +37,8 @@
 #include "qv4objectproto_p.h"
 #include "qv4mm_p.h"
 #include "qv4scopedvalue_p.h"
+#include "qv4alloca_p.h"
+
 #include <QtCore/qnumeric.h>
 #include <QtCore/qmath.h>
 #include <QtCore/QDateTime>
